### DISCUSSION TITLE
In which folder should i put questions?

### DISCUSSION THREAD

**Discussion by soumyaranjan-panda** [Spectrum-CETB/Spectober_Fest#51] [Q&A] [answered]
In which folder should I put the other questions' solutions, in my name folder? if yes then do I have to add another readme file there?

![Screenshot 2022-10-03 185532](https://user-images.githubusercontent.com/84384924/193589161-3096e68a-bb9b-4473-b539-6296a2f40cd2.png)

**Comment by soumyaranjan-panda** [comment]
what do I have to do here? delete the anagram file and keep only the maximum sum file. and then pr?


![Screenshot 2022-10-03 192952](https://user-images.githubusercontent.com/84384924/193596540-c7aaa72f-a765-4907-974c-dde57f5e0bde.png)

**Comment by Aditya-150** [comment, marked as answer]
We wanted to have separated folder for each question solution for each contributor . But we did a mistake in mentioning correctly in the `README.md` file. As you can see other contributors are putting their file of different questions in same folder of their name. So you can also do that.
It was a mistake from our end so we are accepting your solution in the same folder. At least we suggest you write readme for each problem solution. You can put different solution readme in a single readme of your folder. Just mention about the solutions differently in that  readme file.
Sorry for the late reply and mistake in main readme which wasted your valuable time.

Thank You❤️
I hope doubt is cleared. Please `mark as answer` if it was helpful🙂

**Comment by soumyaranjan-panda** [comment]
For every new pr, I used to delete the file that was already there and make a new file for the new one. Like I had a file maximum _sum.java I deleted it and added sort_01.java. Hence the maximum file is not saved. Now I'm pr again for the maximum_sum.java file will that be a problem? i

**Comment by Aditya-150** [comment]
You can add your solutions in the same folder of your name but make separate prs for separate problems solutions. Empty files will not be accepted.

**Reply by Aditya-150** [reply]
> You can add your solutions in the same folder of your name but make separate prs for separate problems solutions.

Once your all prs for that folder have been merged. You can add the file of other question solution making separate pr.
Hope your doubt is cleared now.

**Reply by Aditya-150** [reply]
Have any doubts related to anything you can ask here.

**Comment by soumyaranjan-panda** [comment]
Ok Thanks a lot @Aditya-150 bhaiya

**Reply by Aditya-150** [reply]
> Ok Thanks a lot @Aditya-150 bhaiya

Thanks for providing the screenshots🙂 it helped to understand your exact doubt.
